Source/JavaScriptCore/Scripts/generate-combined-inspector-json.py

```diff
@@ -66,6 +69,10 @@
         sys.stderr.write("File \"%s\" does not contain valid JSON:\n" % file)
         raise
 
+    for flag in known_flags:
+        if flag in string:
+            used_flags.add(flag)
+
     print(string.rstrip())
-print("]}")
+print("], \"flags\": \"" + " ".join(used_flags) + "\"}")
```

Comment:
Other than a bit of extra parsing overhead on the other end, is there a reason to not have this been a real array of strings? (I guess we do need it to be a space-separated string later when we call to perl, but it seems odd that implementation detail would be present in the JSON files)

Reply:
no real reason other than that at the other end we use it as a raw string, so it just means moving the `"".join(...)`

i personally prefer this approach cause it better matches the way it's actually used (and therefore implicitly implies to future engineers "hey dont use this as an array, pass it as one big thing")